Makefile

```diff
@@ -29,44 +29,56 @@ endif
 
 .DEFAULT_GOAL := $(NAME)
 
-.build/deps-build-go.ok:
-	@mkdir -p .build
-	$(GO) get -u github.com/golang/dep/cmd/dep
-	$(GO) get -u github.com/jteeuwen/go-bindata/...
-	$(GO) get -u github.com/elazarl/go-bindata-assetfs/...
+.build-deps/deps-build-go.ok:
```

Comment:
why .build-deps ?

Reply:
Can you copy the promu in the changelog to make that more obvious ?

Reply:
I updated the PR. let me know if by changelog you meant commit message. Thx

Reply:
Yep, commit message :)